protostar-rust/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
+use pyo3::exceptions::RuntimeError;
+
+use anyhow::Context;
+use std::collections::HashMap;
+
+use cairo_lang_protostar::test_collector::collect_tests;
+use cairo_lang_runner::{SierraCasmRunner, RunResultValue};
+
+fn run_result_value_to_string(run_result: RunResultValue) -> String {
+  match run_result {
+    RunResultValue::Success(data) => {
+      return format!("Success {:?}", data);
+    }
+    RunResultValue::Panic(data) => {
+        
+      return format!("Panic {:?}", data);
+    }
+  }
+}
+
+fn internal_run_tests(input_path: &str) -> anyhow::Result<Vec<(String, String)>> {
+  let (sierra_program, test_configs) = collect_tests(&input_path.to_owned(), None, None, None)?;
```

Comment:
Not sure if `test_configs` is a best name for this. Wouldn't `test_case` or something like that be better?

Reply:
It is the name used by cairo test runner

Reply:
The type is `Vec<TestConfig>` so I find this name pretty intuitive. I'm resolving this, if you still have a strong opinion here, please unresolve and state your arguments.